grails-app/controllers/org/pih/warehouse/api/SelectOptionsApiController.groovy

```diff
@@ -0,0 +1,38 @@
+/**
+* Copyright (c) 2012 Partners In Health.  All rights reserved.
+* The use and distribution terms for this software are covered by the
+* Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+* which can be found in the file epl-v10.html at the root of this distribution.
+* By using this software in any fashion, you are agreeing to be bound by
+* the terms of this license.
+* You must not remove this notice, or any other, from this software.
+**/
+package org.pih.warehouse.api
+
+import grails.converters.JSON
+import org.pih.warehouse.core.SelectOptionsService
+
+class SelectOptionsApiController {
+
+    SelectOptionsService selectOptionsService;
+
+    def glAccountOptions = {
+        def glAccounts = selectOptionsService.getGlAccountsOptions()
```

Comment:
@awalkowiak What do you think about using the generic API service here?

```
def glAccountOptions = { 
    def glAccounts = genericApiService.getList(GlAccount.class, [:]).collect {
            [id: it.id, label: "${it.code} - ${it.name}"]
        }
    }
    render([data: glAccounts] as JSON)
}
```

Reply:
Yes, if we can get rid of adding one another redundant service, then this should be good. cc: @alannadolny